src/main/java/org/broadinstitute/consent/http/resources/Resource.java

```diff
@@ -121,6 +122,8 @@ private interface ExceptionHandler {
                 Response.status(Response.Status.BAD_REQUEST).type(MediaType.APPLICATION_JSON).entity(new Error(e.getMessage(), Response.Status.BAD_REQUEST.getStatusCode())).build());
         dispatch.put(BadRequestException.class, e ->
                 Response.status(Response.Status.BAD_REQUEST).type(MediaType.APPLICATION_JSON).entity(new Error(e.getMessage(), Response.Status.BAD_REQUEST.getStatusCode())).build());
+        dispatch.put(MalformedJsonException.class, e ->
+                Response.status(Response.Status.BAD_REQUEST).type(MediaType.APPLICATION_JSON).entity(new Error(e.getMessage(), Response.Status.BAD_REQUEST.getStatusCode())).build());
```

Comment:
got this exception, and it sent back as a 500 - added this here since it seems to be an indicator that the user sent bad Json.